compiler/src/iree/compiler/Codegen/Common/GPU/GPUAllocPrivateMemoryForDPSOps.cpp

```diff
@@ -56,6 +56,9 @@ void GPUAllocPrivateMemoryForDPSOpsPass::runOnOperation() {
   // dimensions).
   SmallVector<OpOperand *> worklist;
   funcOp.walk([&](DestinationStyleOpInterface dpsOp) {
+    if (!dpsOp.hasPureTensorSemantics()) {
+      return;
+    }
     for (int idx = 0; idx < dpsOp.getNumDpsInits(); ++idx) {
       OpOperand *value = dpsOp.getDpsInitOperand(idx);
       if (!dpsOp->getResult(idx).use_empty()) {
```

Comment:
Can update this too while you're here
```suggestion
      if (!dpsOp.getTiedOpResult(value).use_empty()) {
```

Reply:
Done, thanks!